xla/stream_executor/numeric_options.h

```diff
@@ -21,15 +21,26 @@ namespace stream_executor {
 // Options that specify the numeric behavior of operations like matrix
 // multiplications and convolutions
 struct NumericOptions {
-  NumericOptions(bool require_determinism, bool allow_tf32)
-      : require_determinism(require_determinism), allow_tf32(allow_tf32) {}
+  NumericOptions(bool require_determinism, bool allow_tf32, int grad)
+      : require_determinism(require_determinism), allow_tf32(allow_tf32), grad_flags(grad | GF_Initialized) {}
 
-  NumericOptions() : require_determinism(false), allow_tf32(true) {}
+  NumericOptions() : require_determinism(false), allow_tf32(true), grad_flags(0) {}
 
   // If true, the op must be deterministic
   bool require_determinism;
   // If true, float32 inputs can be rounded to TensorFloat-32 precision
   bool allow_tf32;
+
+  enum GradientFlags {
```

Comment:
Please document the enum you introduced here in code comments. I'm not an expert on StreamExecutor or hipBLAS, but I think I should get an idea what the enum values are used for.